src/main/resources/application.yml

```diff
@@ -48,6 +48,11 @@ spring:
   codec:
     max-in-memory-size: 10MB
 
+  profiles:
+    group:
+      test:
+        - "dev"
```

Comment:
not having :point_up: meant that we didn't get any logging when running the tests